app/api/orders.py

```diff
@@ -270,6 +274,9 @@ def before_delete_object(self, order, view_kwargs):
         elif order.amount and order.amount > 0 and (order.status == 'completed' or order.status == 'placed'):
             raise ConflictException({'source': ''}, 'You cannot delete a placed/completed paid order.')
 
+        # delete the attendees so that the tickets are unlocked.
+        delete_related_attendees_for_order(self, order)
```

Comment:
Here, is this a cascade delete? This will be not be automatically done by the DB?

Reply:
oh, yes. Thanks for this. Still need it for cancelled order though. Updating the PR.